artifacts/deployment.json

```diff
@@ -21,5 +21,28 @@
     "s5USDAAPL": "0xb390CD29EA45066E06623480D1b235f50E6618c8",
     "pool": "0x8b205c597602ebf442857D4714d996B343fFa20c",
     "pool2": "0x6582204488d330ffAf464592179936EA1E8A3c1f"
+  },
+  "development": {
```

Comment:
those will be different for everyone, maybe we need to restructure the format of those files to make one network per file? `artifacts/kovan/deployment.json` and `artifacts/development/deployment.json` etc
and we can gitignore `artifacts/deployment`

Reply:
I was thinking about that but travis `cd subgraph & yarn build` will fail